cluster/network.go

```diff
@@ -905,7 +905,7 @@ func (c *Cluster) deployListener(ctx context.Context, host *hosts.Host, portList
 
 	logrus.Debugf("[network] Starting deployListener [%s] on host [%s]", containerName, host.Address)
 	if err := docker.DoRunContainer(ctx, host.DClient, imageCfg, hostCfg, containerName, host.Address, "network", c.PrivateRegistriesMap); err != nil {
-		if strings.Contains(err.Error(), "bind: address already in use") {
+		if strings.Contains(err.Error(), ": address already in use") {
```

Comment:
Is leaving `: ` significant here? It doesn't necessarily hurt but looks quite off.

```suggestion
		if strings.Contains(err.Error(), "address already in use") {
```

Reply:
No significant impact & happy to remove colon if team prefers/think it's best!

I am still fairly new to Docker Engine v28 Spec, so wanted to be a bit cautious incase docker had decided to change another error message to now include "address already in use", so keep the `: ` to try and ensure we only catch the expected error and not a random one.

However just had another quick scan at the Docker Engine Code and I can't see any other ways for error messages using "address already in use" 👍🏻

Reply:
@Faulkner-Ned , agree - it's questionable what is the best option. I'll let other reviewers chime in on that.